RELEASE.md

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: major
+
+Updates module layout
+Adds container definition module
+Adds firelens logging module (Wellcome specific)
```

Comment:
A little more explanation or detail would be nice! e.g. if I was using modules XYZ in the old structure, what should I use now instead?